### PR TITLE
[plugins] Fix some PYLint and PEP8 issues

### PR DESCRIPTION
**Pull request by pponnuvel**
Following #3472 and #3505, I've fixed several pylint and PEP8 issues. Ran pylint, pep8 and flake8 over the changes.

My plan is to have a series of small PRs like this to address the rest of the code under `sos/report/plugins` to start with.

This PR addresses all such warnings for file names starting with alphabets [abcde] except [C0114](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/missing-module-docstring.html), [C0115](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/missing-class-docstring.html), and [C0209](https://pylint.pycqa.org/en/latest/user_guide/messages/convention/consider-using-f-string.html).

C0114 and C0115 requires a bit of knowledge about each of plugins to be able to add meaningful docstrings (I don't want to add `""" This is Ansible Plugin """` for example). C0209 is rather large and we may be able to automate it. These three can addressed separately later.

Please review and let me know if you've any concerns. I've also added some inline comments for each of review.
If this is OK, I'll proceed with next batch of files.

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [x] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

File: sos/report/plugins/activemq.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def postproc(self):
 class RedHatActiveMq(ActiveMq, RedHatPlugin):
 
     def setup(self):
-        super(RedHatActiveMq, self).setup()
+        super().setup()
```
Comment (pponnuvel) [inline sos/report/plugins/activemq.py:45]: Addresses R1725: super-with-arguments

File: sos/report/plugins/alternatives.py [hunk 2]
```diff
@@ -15,6 +15,8 @@ class Alternatives(Plugin):
 
     short_desc = 'System alternatives'
     plugin_name = 'alternatives'
+    alternatives_cmd = None
```
Comment (pponnuvel) [inline sos/report/plugins/alternatives.py:18]: class members defined later in the code. Addresses E1101.

File: sos/report/plugins/arcconf.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 from sos.report.plugins import Plugin, IndependentPlugin
 
 
-class arcconf(Plugin, IndependentPlugin):
+class ArcConf(Plugin, IndependentPlugin):
```
Comment (pponnuvel) [inline sos/report/plugins/arcconf.py:18]: CamelCase

Comment (pponnuvel) [inline sos/report/plugins/arcconf.py:18]: CapWords for class names: https://peps.python.org/pep-0008/#class-names

File: sos/report/plugins/ebpf.py [hunk 4]
```diff
@@ -22,27 +22,29 @@ class Ebpf(Plugin, IndependentPlugin):
     ]
 
     def get_bpftool_prog_ids(self, prog_json):
+        """ Collect the list of program IDs """
         out = []
         try:
             prog_data = json.loads(prog_json)
-        except Exception as e:
-            self._log_info("Could not parse bpftool prog list as JSON: %s" % e)
+        except Exception as err:  # pylint: disable=broad-except
```
Comment (pponnuvel) [inline sos/report/plugins/ebpf.py:29]: Address [C0200](https://pylint.readthedocs.io/en/stable/user_guide/messages/convention/consider-using-enumerate.html).

I've verified that the changes are functionally equivalent.

File: sos/report/plugins/ebpf.py [hunk 5]
```diff
@@ -22,27 +22,29 @@ class Ebpf(Plugin, IndependentPlugin):
...
             map_data = json.loads(map_json)
-        except Exception as e:
-            self._log_info("Could not parse bpftool map list as JSON: %s" % e)
+        except Exception as err:  # pylint: disable=broad-except
+            self._log_info("Could not parse bpftool map list: %s" % err)
             return out
-        for item in range(len(map_data)):
-            if "id" in map_data[item]:
-                out.append(map_data[item]["id"])
+        for _, item in enumerate(map_data):
```
Comment (pponnuvel) [inline sos/report/plugins/ebpf.py:45]: Same as [above](https://github.com/sosreport/sos/pull/3530/files#r1493372882)

File: sos/report/plugins/clear_containers.py [hunk 6]
```diff
@@ -73,9 +77,9 @@ def setup(self):
             self.limit = None
 
         self.add_cmd_output('{} cc-env'.format(self.runtime))
-        self.attach_cc_config_files()
+        self.collect_cc_config_files()
```
Comment (pmoravec) [inline sos/report/plugins/clear_containers.py:80]: There is a merge conflict with https://github.com/sosreport/sos/commit/57d21346b2952bf7c995eaeec4a98e1ae77c6308 .

File: sos/report/plugins/azure.py [hunk 7]
```diff
@@ -8,8 +8,8 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.report.plugins import Plugin, UbuntuPlugin, RedHatPlugin
 import os
+from sos.report.plugins import Plugin, UbuntuPlugin, RedHatPlugin
```
Comment (pmoravec) [inline sos/report/plugins/azure.py:12]: Why this change (repeated more times), please? I don't insist on either ordering of imports, just curious.

File: sos/report/plugins/clear_containers.py [hunk 8]
```diff
@@ -21,8 +21,10 @@ class ClearContainers(Plugin, IndependentPlugin):
     runtime = 'cc-runtime'
     packages = (runtime,)
     services = ('cc-proxy',)
+    limit = None
```
Comment (pmoravec) [inline sos/report/plugins/clear_containers.py:24]: I had to search in code what is the purpose of the variable. When we touch it, maybe it makes sense to rename it to e.g. `loglimit`? Or is the current naming sufficient?

**Comment by pmoravec** [comment]
ACK up to a few minor comments. Nice!

File: sos/report/plugins/clear_containers.py (hunk 8 above)
Comment (pponnuvel) [reply sos/report/plugins/clear_containers.py]: `loglimit` seems better - renamed.

File: sos/report/plugins/azure.py (hunk 7 above)
Comment (pponnuvel) [reply sos/report/plugins/azure.py:12]: For addressing [C0411 - wrong-import-order](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/wrong-import-order.html).

https://peps.python.org/pep-0008/#imports

> Imports should be grouped in the following order:
>
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.

File: sos/report/plugins/clear_containers.py (hunk 6 above)
Comment (pponnuvel) [reply sos/report/plugins/clear_containers.py]: Ack, fixed

File: sos/report/plugins/azure.py (hunk 7 above)
Comment (pmoravec) [reply sos/report/plugins/azure.py:12]: Ah I see, thanks for explanation.

**Review by pmoravec — APPROVED** [review]

**Review by TurboTurtle — APPROVED** [review]
